scripts/run-django-dev.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# This script runs the django app, waiting on the react applications to build
+
+health_urls=(
+    "http://watch:8052/health"
+)
+wait_time=300
+
+if [[ $NODE_ENV == "production" ]] ; then
```

Comment:
When $NODE_ENV is development, we use webpack-dev-server, which serves the built code from memory, not from filesystem.

If you're running in production locally, we need to wait for the build to finish before starting django. (In particular, I believe, before running collectstatic?).

Modeled after @rhysyngsun 's work in https://github.com/mitodl/mitxonline/pull/628